test/functional/core/job_spec.lua

```diff
@@ -274,10 +273,11 @@ describe('jobs', function()
   end)
 
   it('can omit options', function()
-    if helpers.pending_win32(pending) then return end
     neq(0, nvim('eval', 'delete(".Xtestjob")'))
-    nvim('command', "call jobstart(['touch', '.Xtestjob'])")
-    nvim('command', "sleep 100m")
+    local touch_cmd = iswin() and "'Out-File -encoding ASCII .Xtestjob'"
+                              or "['touch', '.Xtestjob']"
```

Comment:
I'm surprised Lua continues the line. Even if it does, should put parens around it to be explicit. 

    local touch_cmd = (iswin() ...
                              or ...)